### DISCUSSION TITLE
v2.2.0

### DISCUSSION THREAD

**Discussion by Victor-IX** [Victor-IX/Blender-Launcher-V2#124] [Announcements]
# What's Changed

## 💡 Features:
- Add a message to notify the existence of tray icon feature for Windows version
- Library Folder can now be relative to the Launcher location
- Improve Column Ordering, it's now possible to have ascending and descending ordering
- Add option to show achieved build for daily and Experimental Download

## 🛠️ Fixes:
- Fix a build ordering bug in the download tab
- Fix a bug caused by specific timezone on Windows

## ⚙️ Others:
- Documentation Update
- Improve GitHub Action to create automatic build
- UI improvement in the Settings menu
- Blender Launcher has been added to the Archlinux User Repository (maintained by @sakarimo)

<hr /><em>This discussion was created from the release <a href='https://github.com/Victor-IX/Blender-Launcher-V2/releases/tag/v2.2.0'>v2.2.0</a>.</em>

**Comment by DoUhonestlyCare** [comment]
Windows Security blocked a threat called Trojan:Script/Wacatac.B!ml and it was from when I tried to update the launcher. It said it was this file: F:\Blender_Launcher_v1.16.1_Windows_x64\.temp\Blender_Launcher_v2.2.0_Windows_x64.zip

**Reply by Victor-IX** [reply]
That happens with new releases; it has to do with our build system(PyInstaller). You have nothing to worry about. I don't see Windows Defender tagging the last version as malicious on virustotal. Maybe it fixed itself, but to be sure, I will make a manual review submission to Microsoft to have the version whitelisted.
